content/docs/template_reference.md

```diff
@@ -0,0 +1,104 @@
+title = "Template Quick Reference"
+description = "A catalog of values and functions for your templates"
+date = "2022-01-27T02:57:20Z"
+---
+
+This page provides a quick reference for things you can use in your templates, like values and functions.
+
+## Values Reference
+
+The following values are available in the template. This is formatted in YAML for readability. The three top-level objects are:
+
+- *page:* The content for the current page
+- *site:* Info about the entire site
+- *env:* Information from the browser or the request, presented as environment variables.
+
+To reference a particular value, use dotted notation. For example, `page` has a `head`, which has the page's `title`.
+To access the title, use `{{ page.head.title }}`.
+
+
+```yaml
+# Page holds data specific to the page that matches the URI. For example, the URI /foo loads page data from /content/foo.md
+page:
+    # The info from the TOML head:
+    head:
+        date: "2022-01-11T20:08:47Z" # In templates, use `date_format "%B %m, %Y" page.head.date` to format
+        title: "WebAssembly Language Support Matrix" # The H1 title
+        description: "Tracking the programming languages that compile to WebAssembly (Wasm)."
+        template: "page"
+        tags: ["webassembly", "programming languages", "javascript", "python", "rust", "dotnet", "ruby"]
+
+        extra: # Remember that everything in extra is optional, so wrap in #if
+            author: "Fermyon Staff"
+            author_page: "/author/"
+            image: "/static/some/thing.png"
+            last_modified: "2022-01-11T20:08:47Z" # Only if the page is modified
+    body: "The HTML content"
+    published: true # Set to false if either manually disabled by author or if `date` is in the future
+
+# Site holds sitewide data
+site:
+    # Info keeps all of the information from `site.toml`
+    info:
+        title: "The site title"
+        base_url: "https://something" # Note no trailing slash
+        about: "Pithy site summary"
+        extra:
+            your_defined_field: "Value"
+
+    pages: # Metadata for every page on the site. Note that there is no guarantee that `body` will be set.
+        "/foo":
+            head:
+                date: "2022-01-11T20:08:47Z"
+                title: "Some title"
```

Comment:
Is `description` available as well here?

Reply:
Fixed